src/gfn/examples/estimators.py

```diff
@@ -78,4 +79,16 @@ def to_probability_distribution(
         logits = module_output
         logits[~states.backward_masks] = -float("inf")
 
-        return Categorical(logits=logits)
+        return UnsqueezedCategorical(logits=logits)
```

Comment:
Will we ever need `SqueezedCategorical` is my question -- if we need the unsqueeze for broadcasting, I think we could just overwrite this name `Categorical` from `torch.distributions` with the one specific to `gfn` so that the user never accidentally uses the vanilla `Categorical`?

Reply:
I'm not sure what you mean by "overwriting the name", do you mean just rename `UnsqueezedCategorical` to `Categorical` ? 
Btw the user should *never* have to use Categorical or UnsqueezedCategorical, for discrete environments, given that the distribution is abstracted away when using `DiscretePFEstimator` or `DiscretePBEstimator` (as in the README example). 
The user would need to write their own distributions for custom environments only (as what I'm doing in this PR [although not completely done] with `Box`)

Reply:
Yes that's what I meant by "overwriting". I was assuming that a major use case of the lib would be for people to implement their own envs (since that's going to be the most domain-specific implementation piece) but I also take your point that the specific distributions are going to be based on the specific environments, so we shouldn't automagically overwrite any of the distribution classes.